be/src/connector/connector.h

```diff
@@ -64,6 +64,8 @@ class DataSource {
     }
     void set_read_limit(const uint64_t limit) { _read_limit = limit; }
     Status parse_runtime_filters(RuntimeState* state);
+    // for stream seek offset
+    virtual Status set_offset(int64_t table_version, int64_t changelog_id) { return Status::OK(); }
```

Comment:
Maybe we can abstract the `offset` if we add more stream connectors(eg. kafka) in the future